src/unwind.c

```diff
@@ -20,6 +20,15 @@
 #include <sys/user.h>
 
 /*
+ * some libelf implementations do not provide ELF_C_READ_MMAP
+ */
+#if HAVE_DECL_ELF_C_READ_MMAP
+#define TBSTACK_ELF_C_READ_MMAP ELF_C_READ_MMAP
```

Comment:
Maybe better name it like `TBSTACK_ELF_C_READ` ?

Reply:
Done, fixed & squashed.